### PR TITLE
Translating to Russian language

### PR DESCRIPTION
**Pull request by edgiru**
a translation was made, please add it to the application. Perhaps in due course there will be edits I will fix on-line.

### CODE REVIEW

File: mobile/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    <string name="mainmenu_openhab_clearcache">Очистить кеш картинок</string>
+    <!-- App settings strings -->
+    <string name="settings_connection_title">Параметры Подключения</string>
+    <string name="settings_display_title">Параметры Дисплея</string>
+    <string name="settings_misc_title">Другие параметры</string>
+    <string name="settings_openhab_url">URL адрес локального openHAB</string>
+    <string name="settings_openhab_url_summary">Ручная конфигурация openHAB URL. Отключает автоопределение, если настроен.</string>
+    <string name="settings_openhab_alturl">URL-адрес удаленного openHAB</string>
+    <string name="settings_openhab_alturl_summary">URL-адрес удаленного openHAB (используется, когда локальный openHAB недоступен
+)</string>
```
Comment (mueller-ma) [inline mobile/src/main/res/values-ru/strings.xml]: Please remove the linebreak here

File: mobile/src/main/res/values-ru/strings.xml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    <string name="mtm_accept_cert">Принять неизвестный Сертификат?</string>
+    <string name="mtm_decision_always">Всегда</string>
+    <string name="mtm_decision_once">Один раз</string>
+    <string name="mtm_decision_abort">Отмена</string>
+    <string name="mtm_notification">Проверка сертификата</string>
+    <string name="settings_openhab_none">Нет</string>
+    <!-- theme -->
+    <string name="theme_value_light">светлая</string>
+    <string name="theme_value_dark">темная</string>
+    <string name="theme_value_black">чёрная</string>
```
Comment (mueller-ma) [inline mobile/src/main/res/values-ru/strings.xml]: Please remove the three "theme_value_..." strings

File: mobile/src/main/res/values-ru/strings.xml [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    <string name="theme_name_black">Чёрная/AMOLED</string>
+
+    <!-- nfc toggle -->
+    <string name="nfc_action_on">Вкл</string>
+    <string name="nfc_action_off">Выкл</string>
+    <string name="nfc_action_toggle">Переключатель</string>
+
+    <!-- drawer -->
+    <string name="drawer_open">Sitemap drawer opened</string>
+    <string name="drawer_close">Sitemap drawer closed</string>
```
Comment (mueller-ma) [inline mobile/src/main/res/values-ru/strings.xml]: Not translated

File: mobile/src/main/res/values-ru/strings.xml [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+    <!-- About -->
+    <string name="about_title">О программе</string>
+    <string name="about_license_title">Лицензия</string>
+    <string name="about_links">Ссылки</string>
+    <string name="about_links_list"><![CDATA[\u00B7 <a href="https://github.com/openhab/openhab.android">openHAB App source code</a><br><br> \u00B7 <a href="https://github.com/openhab/openhab.android/issues">Report a problem</a><br><br> \u00B7 <a href="http://docs.openhab.org/">openHAB Documentation</a><br><br> \u00B7 <a href="https://community.openhab.org/">Community Forum</a>]]></string>
```
Comment (mueller-ma) [inline mobile/src/main/res/values-ru/strings.xml]: Not translated

File: mobile/src/main/res/values-ru/strings.xml (hunk 2 above)
Comment (mueller-ma) [reply mobile/src/main/res/values-ru/strings.xml]: They become obsolete when #350 is merged

**Comment by mueller-ma** [comment]
You also removed "theme_name_...". Those three should be included.

**Comment by mueller-ma** [comment]
@digitaldan Can be merged

**Comment by digitaldan** [comment]
Thanks!

**Pull request merged**